test/functionalTest/cases/1316_create_subscription/post_subs_expr_q_not_equal_single_number.test

```diff
@@ -0,0 +1,235 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST Subscriptions V2 q expression (not equal single number)
+
+--SHELL-INIT--
+dbInit CB
+# FIXME P10: replace with just 'brokerStart CB' once cache part gets implemented
+brokerStart CB 0 IPv4 -noCache
+accumulatorStart
+
+--SHELL--
+
+#
+# 01. Create subscription with q: status!=22
```

Comment:
This notifies only for number. As you suggest, in order to be based in string it should use status!='22'. You are right: this is not implemented (as far as I understand looking to qStringFilters function) and is the root cause of bugs likes this one https://github.com/telefonicaid/fiware-orion/issues/1595

However, ' should work to avoid vectors, e.g:

```
colour=='black,white','red,blue'
```

I'd include some tests for that.